PoSSapi/src/Application/Ingredients/Commands/UpdateIngredientCommandValidator.cs

```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
```

Comment:
Maybe put this file to a `Validators` folder? We're doing this elsewhere in the project.